.pre-commit-config.yaml

```diff
@@ -34,7 +34,7 @@ repos:
       - id: mypy
         exclude: ^semgrep\/tests\/.+$|^setup.py$|^scripts\/.+$|^parsing-stats\/.+$|^perf\/.+$$
         args: [--config, mypy.ini, --show-error-codes]
-        additional_dependencies:
+        additional_dependencies: &mypy-deps
```

Comment:
ohoh, yaml anchor master!
Look at that @emjin !

Reply:
Tada